### DISCUSSION TITLE
Apollo Save Tool v1.6.5

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps3#77] [General]
### Added

* New save-game cheats
  * Final Fantasy XIII
  * Ratchet & Clank Future: A Crack in Time
  * Resident Evil 4 HD
  * Resident Evil 6
  * Resident Evil Revelations 2
  * Ryu ga Gotoku Kiwami
  * Tales of Zestiria

### Fixed

* Fixed Online DB URL

### Misc

* Use Apollo-lib (shared with Apollo PS4)

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps3/releases/tag/v1.6.5'>Apollo Save Tool v1.6.5</a>.</em>

**Comment by Kenifornication** [comment]
Would you consider adding some sort of real-time save backup feature to Apollo?

**Reply by bucanero** [reply]
> Would you consider adding some sort of real-time save backup feature to Apollo?

that would be a completely different thing to handle (lots of work), and I'm currently working on other platforms so any updates on PS3 are just for maintenance or bug-fix.
